distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -0,0 +1,27 @@
+# Configures the thread pools of the Mule Runtime
+# Certain attributes support expressions. Those are arithmetic expressions (+, -, / or * operators supported only) that are
+# evaluated when the runtime starts.
+# Those expressions may be applied to numbers (integers or decimals) or any of the following variables:
+# * C: the number of processors available to the Java virtual machine.
+
+# The maximum time to wait until all tasks in all the runtime thread pools have completed execution when stopping the
+# scheduler service.
+SchedulerService.gracefulShutdownTimeoutSeconds=15
+
+# The number of threads to keep in the cpu_lite pool, even if they are idle.
+# Supports Expressions
+SchedulerService.cpuLight.threadPoolSize=2*C
+
+# The number of threads to keep in the I/O pool.
+# Supports Expressions
+SchedulerService.io.coreThreadPoolSize=C
+# The maximum number of threads to allow in the I/O pool.
+# Supports Expressions
+SchedulerService.io.maxThreadPoolSize=C*C
+# When the number of threads in the I/O pool is greater than SchedulerService.io.coreThreadPoolSize, this is the maximum
+# time (in seconds) that excess idle threads will wait for new tasks before terminating.
+SchedulerService.io.threadKeepAliveSeconds=30
```

Comment:
Isn't this best exposed as `ms`, else smallest timeout is 1s and resolution is limited.  (Also can unit go in comment, rather than making property name longer?)